src/sparseml/pytorch/torchvision/train.py

```diff
@@ -408,7 +409,14 @@ def collate_fn(batch):
     if args.distributed and args.sync_bn:
         model = torch.nn.SyncBatchNorm.convert_sync_batchnorm(model)
 
-    criterion = nn.CrossEntropyLoss(label_smoothing=args.label_smoothing)
+    min_torch_version_with_label_smoothing = version.parse("1.10")
+    label_smoothing_supported = (
+        version.parse(torch.__version__) >= min_torch_version_with_label_smoothing
+    )
+    if label_smoothing_supported:
```

Comment:
very optional nit would be to just inline all of this with the additional values - i think upon reading the whole if statement it would be clear that label_smoothing is not supported with other versions - especially if we add the raise error

Reply:
Thank you, it does look clearer